cloudcix_primitives/storage_hyperv.py

```diff
@@ -1,19 +1,345 @@
+
+            ret = rcc.run(payloads['dismount_storage_file'])
+            if ret["channel_code"] != CHANNEL_SUCCESS:
+                return False, fmt.channel_error(ret, f"{prefix+4}: " + messages[prefix + 4]), fmt.successful_payloads
+            if ret["payload_code"] != SUCCESS_CODE:
+                return False, fmt.payload_error(ret, f"{prefix+5}: " + messages[prefix + 5]), fmt.successful_payloads
+            fmt.add_successful('dismount_storage_file', ret)
+
+        return True, "", fmt.successful_payloads
+
+    status, msg, successful_payloads = run_host(host, 3020, {})
+    if status is False:
+        return status, msg
+
+    return True, messages[1000]
+
+def scrub(
+    host: str,
+    domain_path: str,
+    storage: str,
+):
+    """
+    description:
+        Removes <domain_path><storage> file on the given Host <host>.
+
+    parameters:
+        host:
+            description: The dns or ipadddress of the Host on which this storage_hyperv is scrubbed
+            type: string
+            required: true
+        domain_path:
+            description: The location or directory path where this storage_hyperv is removed
+            type: string
+            required: true
+        storage:
+            description: The name of the storage_hyperv to be removed
+            type: string
+            required: true
+    return:
+        description: |
+            A tuple with a boolean flag stating the remove was successful or not and
+            the output or error message.
+        type: tuple
+    """
+    # Define message
+    messages = {
+        1100: f'Successfully removed storage image {storage} from {domain_path} on Host {host}.',
+        1101: f'Storage image {storage} from {domain_path} does not exist on Host {host}',
+
+        3121: f'Failed to connect to the Host {host} for the payload read_storage_file: ',
+        3122: f'Failed to connect to the Host {host} for the payload remove_storage_file: ',
+        3123: f'Failed to run remove_storage_file payload on the Host {host}. Payload exited with status '
+    }
+
+    def run_host(host, prefix, successful_payloads):
+        rcc = SSHCommsWrapper(comms_ssh, host, 'robot')
+        fmt = HostErrorFormatter(
+            host,
+            {'payload_message': 'STDOUT', 'payload_error': 'STDERR'},
+            successful_payloads
+        )
+
+        payloads = {
+            'read_storage_file': f'Get-VHD -Path {domain_path}{storage}',
+            'remove_storage_file': f'Remove-Item -Path {domain_path}{storage} -Recurse -Force -Confirm:$false',
+        }
+
+        ret = rcc.run(payloads['read_storage_file'])
+        if ret["channel_code"] != CHANNEL_SUCCESS:
+            return False, fmt.channel_error(ret, f"{prefix+1}: " + messages[prefix+1]), fmt.successful_payloads
+        if ret["payload_code"] != SUCCESS_CODE:
+            return False, fmt.payload_error(ret, f"1101: " + messages[1101]), fmt.successful_payloads
```

Comment:
If the payload for "read_storage_file" does not return 0 as the code, this indicates the storage does not exist and has already been scrubbed, 

As this is the scrub verb, I do not think that as an error/failure of the request